README.md

```diff
@@ -179,6 +179,7 @@ appium:chromeOptions | A mapping, that allows to customize chromedriver options.
 Capability Name | Description
 --- | ---
 appium:disableSuppressAccessibilityService | Being set to `true` tells the instrumentation process to not suppress accessibility services during the automated test. This might be useful if your automated test needs these services. `false` by default
+appium:disableWindowAnimation | Disables the window animation for Android device under test. `true` by default
```

Comment:
Perhaps it makes sense to describe in more details how enabling or disabling animations affects Espresso tests

Reply:
Added link for google espresso testing recommendation https://developer.android.com/training/testing/espresso/setup#set-up-environment